content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -11,7 +11,8 @@ weight: 30
 
 {{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
 
-PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25. For more information on the deprecation,
+PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25. It has been replaced by
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/). For more information on the deprecation,
```

Comment:
This change looks like it might belong in a separate PR @SpecialYang ?

Reply:
This change has be merged already in repo. It appears here due to my mistake git op.